docs/niscope/session.rst

```diff
@@ -148,6 +154,8 @@ niscope.Session
    +-----------------------------------------------------+---------------------------------------+
    | :py:attr:`logical_name`                             | str                                   |
    +-----------------------------------------------------+---------------------------------------+
+   | :py:attr:`manual_configuration_enabled`             | :py:data:`BoolEnableDisable`          |
```

Comment:
Looks like this used to be p2p_manual_configuration_enabled... so we should remove?

Reply:
Do we explicitly remove it or leave it in case P2P ever adds Python support?

Reply:
So far we've removed everything related to P2P from the APIs. I think we should continue doing so until/if P2P adds Python support.
As it is it's just noise and confusion.

Reply:
Removed peer to peer attributes for both NI-SCOPE and NI-FGEN